scss/components/table.scss

```diff
@@ -192,21 +203,40 @@ $block: #{$fd-namespace}-table;
         overflow-x: scroll;
         padding-left: 200px;
 
-        td {
+        .#{$block}__cell {
             min-width: 200px;
         }
     }
 
     &--no-borders {
         border: none;
-        & tbody tr {
```

Comment:
I doesn't look like the block is working as expected, look at the `--no-borders` example, there shouldn't be any borders between the rows:
<img width="1430" alt="Screen Shot 2019-08-21 at 11 55 27 AM" src="https://user-images.githubusercontent.com/29607818/63460165-d4c66680-c40a-11e9-8ce5-296efc75ba49.png">